### DISCUSSION TITLE
Please add support for the HTML details tag

### DISCUSSION THREAD

**Discussion by muru** [community/community#2822] [Mobile]
Currently the mobile app doesn't support the HTML `<details>` tag. The summary and (supposed to be hidden by default) contents are both shown directly.

The `<details>` tag is used for hiding text behind a summary. It's very useful for including long snippets of logs or command output in a discussion without it dominating the post.

E.g.:

<details>
<summary>Screenshots of a collapsed and expanded details tag</summary>

> ![image](https://user-images.githubusercontent.com/4310271/100825517-9045c980-349b-11eb-9c08-9f2a9265629b.png)
> ![image](https://user-images.githubusercontent.com/4310271/100825544-a0f63f80-349b-11eb-88b5-15740b25bb8e.png)

</details>

However, in the mobile app, both the summary and the content are shown directly. This is particularly annoying, since the disruptive effect of such long blocks of logs is exacerbated by the reduced display area of phones.

**Comment by candyho** [comment]
Hey, thanks for flagging and apologise for the inconvenience caused. `<details>` tag isn't supported on the Android app yet, are you on Android/iOS? I have added this to our improvement backlog. Will keep you posted when we land on a solution.

**Reply by Marcono1234** [reply]
Looks like this was implemented, see https://github.com/orgs/community/discussions/3632#discussioncomment-749260
